spec/javascript/setup.js

```diff
@@ -0,0 +1,3 @@
+var jsdom = require('jsdom');
+global.$ = require('jquery')(new jsdom.JSDOM().window);
+
```

Comment:
https://forum.sublimetext.com/t/remove-any-extra-blank-lines-at-end-of-file-on-save/6175/3